src/test/java/LottoMachineTest.java

```diff
@@ -0,0 +1,21 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LottoMachineTest {
```

Comment:
아래와 같이 클래스에도 `@DisplayName` 을 사용할 수 있어요~
```suggestion
@DisplayName("LottoMachine 테스트")
class LottoMachineTest {
```